wandb/sdk/artifacts/artifact.py

```diff
@@ -217,6 +187,48 @@ def __init__(
         # Cache.
         get_artifacts_cache().store_client_artifact(self)
 
+    @staticmethod
+    def _get_artifact_fragment() -> str:
```

Comment:
Nit: it's not clear what an "artifact fragment" is. How about `_get_gql_fragment`?